install/index.php

```diff
@@ -14,7 +14,7 @@
 	$database_password = trim($_POST['database_password']);
 	$database_name = trim($_POST['database_name']);
 
-	if (!$database_host) $errorMessage[] = 'Host name is required';
+	if (!$database_host) $errorMessage[] = _('Host name is required');

```

Comment:
we discussed of this a few minuts ago: no need to translate the install step (for now, we will see later. But I'm not sure it's relevent to translate the installer)